### DISCUSSION TITLE
Full section notification for course enrolment

### DISCUSSION THREAD

**Discussion by GregSherman** [sqrl-planner/sqrl-client#79] [Ideas]
If you have a timetable that’s been made for a while and you go to look at it but a course has been filled up, maybe a notification can tell you and let's you choose another lecture section or ignore it. 

This would be useful during course enrolment if data is updated live so you can quickly see backups, would also be useful to have that as a feature for auto schedules. If a section gets full it'll rearrange everything as little as possible to have a non full schedule.

Let's you get an edge on everyone else who has to scramble for backup schedules, would make sqrl the ideal use during course enrolment.

**Comment by ICPRplshelp** [comment]
the timetable API this program uses does not update as quickly as acorn does, which likely updates one per day

Of course, ttb updates way more frequently (perhaps up to the minute) but it is way harder to scrap

**Reply by galacticglum** [reply]
Moving forward, gator (data service that sqrl is powered by) will be sourcing data from the ttb api rather than the artsci timetable builder — the difficulty isn't so much in scraping the api as it is in migrating our infrastructure to a completely different dataset.
